addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -37,7 +37,8 @@
         <span t-else="" class="fw-bold text-uppercase">Free</span>
     </xpath>
     <xpath expr="//div[hasclass('o_wevent_price_range')]" position="inside">
-        <t t-set="all_prices" t-value="event.event_ticket_ids.mapped('price_reduce')"/>
+        <t t-if="website.show_line_subtotals_tax_selection == 'tax_excluded'" t-set="all_prices" t-value="event.event_ticket_ids.mapped('price_reduce')"/>
+        <t t-else="website.show_line_subtotals_tax_selection == 'tax_included'" t-set="all_prices" t-value="event.event_ticket_ids.mapped('price_reduce_taxinc')"/>
```

Comment:
Should either be t-elif or just t-else without anything.

qweb won't evaluate whatever is set for `t-else`

Reply:
Good to know- thanks :) it should be okay now